decidim/decidim
# Add tabs to Pages

#### :tophat: What? Why?
This PR creates a stand alone space for page topics, by adding the tabbing system as it was agreed in the @decidim/maintainers 's weekly call 
Additionally the Pr fixes the label for "Last notable change" in the static page index. 
This PR does not add pagination for any of the models, does not add additional filtering.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes  #11785

#### Testing
1. Login as admin and visit the admin interface 
2. Click on Pages in the sidebar menu 
3. See the buttons, see the wrong label for Tos
4. Apply patch 
5. Refresh page 
6. See tabs are correctly maintained when navigating through pages
7. See breadcrumb when switching to topics ( adds an extra level "topics" ) 
8. See static page 

### :camera: Screenshots
Before:
![image](https://github.com/decidim/decidim/assets/105683/bf821874-3eb7-49a4-ae6a-2acbacf7de96)

After: 
Topics:
![image](https://github.com/decidim/decidim/assets/105683/ede10c14-178a-4834-9f0b-c886ad22a5d0)
![image](https://github.com/decidim/decidim/assets/105683/a04611c9-96fd-47fb-8dfa-06a324f3efd0)
Pages:
![image](https://github.com/decidim/decidim/assets/105683/74aeaf14-5615-4d2c-9021-a4ea5d245208)
![image](https://github.com/decidim/decidim/assets/105683/daf57bf5-7cb8-4e46-8584-f4e078376669)


:hearts: Thank you!
